hikari/impl/rest.py

```diff
@@ -1035,7 +1035,7 @@ async def create_message(
             content = undefined.UNDEFINED
 
         elif undefined.count(attachment, attachments) == 2 and isinstance(
-            content, (files.Resource, files.RAWISH_TYPES, os.PathLike)
+            content, (files.Resource, files.RAWISH_TYPES, os.PathLike)  # type: ignore[misc]
```

Comment:
what type error are you ignoring here?

Reply:
Forgot to specify it, will add it later when I get time, as I have to rebase all the pr's I have open